lab3/ans/deploy_ans.yaml

```diff
@@ -0,0 +1,303 @@
+version: "3.8"
+
+services:
+  free5gc-upf:
+    container_name: upf
+    image: free5gc/upf:v3.4.2
+    command: bash -c "./upf-iptables.sh && ./upf -c ./config/upfcfg.yaml"
+    volumes:
+      - ./config/upfcfg.yaml:/free5gc/config/upfcfg.yaml
+      - ./config/upf-iptables.sh:/free5gc/upf-iptables.sh
+    cap_add:
+      - NET_ADMIN
+    networks:
+      n3net:
+        aliases:
+          - upf.n3.org
+        ipv4_address: 10.100.3.100
+      n4net:
+        aliases:
+          - upf.n4.org
+      n6net:
+
+
```

Comment:
Add n6gw service to forward the traffic from UPF N6 to internet.

Reply:
Add additional routing rule in n6 gateway container:
```
ip r add {UE_POOL_CIDR} via {UPF_N6_IP}
```
to forward the UE downlink traffic from Data network to UPF.